deeprank_gnn/domain/amino_acid.py

```diff
@@ -1,31 +1,189 @@
-from deeprank_gnn.models.amino_acid import AminoAcid
-from deeprank_gnn.models.polarity import Polarity
+from ..models.amino_acid import AminoAcid

```

Comment:
Relative imports only work within packages, so I added __init__.py file in domain folder so that then domain folder becomes a package.